docs/org/reference/language/functions.org

```diff
@@ -47,3 +47,41 @@ odd (suc n) := even n;
 even zero := true;
 even (suc n) := odd n;
 #+end_example
+
+** Anonymous functions
+
+Anonymous functions, or /lambdas/, are introduced with the syntax:
+
+#+begin_example
+\{pat1 .. patN_1 := clause1;
+  ..
+  pat1 .. patN_M := clauseM}
+#+end_example
+
+An anonymous function just lists all clauses of a function without
+naming it. Any function declaration can be converted to use anonymous
+functions:
+
+#+begin_example
+open import Stdlib.Prelude;
+
+odd : Nat → Bool;
+even : Nat → Bool;
+
+odd := \{
+  zero := false;
+  (suc n) := even n;
+};
+
+even := \{
+  zero := true;
+  (suc n) := odd n;
+};
+#+end_example
+
+** Short definitions
+
+If a function has only one clause, then its definition can written in one line, with the body immediately following the signature:
+#+begin_example
+multiplyByTwo : Nat -> Nat := \{n := 2 * n};
```

Comment:
and here we are using ascii arrow. We should be consistent

Reply:
Done